plaso/cli/helpers/artifact_definitions.py

```diff
@@ -86,11 +96,20 @@ def ParseOptions(cls, options, configuration_object):
       raise errors.BadConfigOption(
           'Unable to determine path to artifact definitions.')
 
+    custom_artifacts_path = getattr(
+        options, 'custom_artifact_definitions_path', None)
+
     registry = artifacts_registry.ArtifactDefinitionsRegistry()
     reader = artifacts_reader.YamlArtifactsReader()
 
     try:
       registry.ReadFromDirectory(reader, artifacts_path)
+      if custom_artifacts_path and os.path.isfile(custom_artifacts_path):
+        registry.ReadFromFile(reader, custom_artifacts_path)
+      elif custom_artifacts_path and not os.path.isfile(custom_artifacts_path):
+        raise errors.BadConfigOption(
+            'No such artifacts filter file: {0:s}.'.format(
```

Comment:
nit: looks like this and the next line could fit on a single line

Reply:
Unfortunately, they do not fit on a single line.